### ISSUE TITLE
after finetune on lerobot ,how to eval :)

### ISSUE THREAD

**Issue by Luo-Zhongwei** [NVIDIA/Isaac-GR00T#108]
as we all know ,there are hundreds of lerobot datasets on huggingface 

for example:
```python
lerobot/metaworld_mt50_push_v2_image
lerobot/libero_object_image
lerobot/libero_goal_image
lerobot/libero_spatial_image
lerobot/libero_10_图像
lerobot/austin_buds
lerobot/xarm_push_medium_replay_image
lerobot/umi_cup_in_the_wild
lerobot/cmu_play_fusion
lerobot/cmu_franka_exploration_dataset
lerobot/berkeley_mvp
lerobot/berkeley_fanuc_manipulation
lerobot/berkeley_autolab_ur5
lerobot/austin_sirius_dataset
lerobot/austin_sailor_dataset
lerobot/aloha_static_ziploc_slide
lerobot/aloha_static_pingpong_test
lerobot/aloha_static_cups_open
lerobot/aloha_static_coffee_new
lerobot/aloha_sim_insertion_scripted_image
lerobot/aloha_sim_insertion_human_image

```
Q1: so you teach me first I need do fine tune on this dataset and it will generate a new GR00T-N1-2B/experiment_cfg/metadata.json ?

Q2: for every single datasets ,Do I need to fine-tune separately for each dataset? all jointly fine tune on hundreds dataset

Q3: how to eval them? for exapmle , I finetune on lerobot/libero_object_image,get a model ,but the model already seen lerobot/libero_object_image dataset, I can't use it to eval, how to solve that?

forgive me propose such foolish problem :)

**Comment by Catchip** [comment]
I also have a question similar to Q3: 
In the VLA field, is it common practice not to split training and validation sets? It seems people evaluate model generalization through performance on unseen tasks rather than by creating validation splits and using validation set performance.

I also noticed there's a "splits" field in the dataset's meta/info.json, but based on my understanding of the code, this field doesn't seem to be actively used. 

Is my observation correct?

**Comment by LukeLIN-web** [comment]
> I also have a question similar to Q3: In the VLA field, is it common not to split training and validation sets? It seems people evaluate model generalization through performance on unseen tasks rather than by creating validation splits and using validation set performance.
> 
> I also noticed there's a "splits" field in the dataset's meta/info.json, but based on my understanding of the code, this field doesn't seem to be effectively utilized.
> 
> Is my observation correct?


True,  In the VLA field, people always evaluate model generalization through performance on unseen tasks rather than by creating validation splits and using validation set performance.
